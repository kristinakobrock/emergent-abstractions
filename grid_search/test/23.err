  0%|                                                                                                     | 0/50 [00:00<?, ?it/s]  6%|#####5                                                                                       | 3/50 [00:00<00:01, 25.09it/s] 12%|###########1                                                                                 | 6/50 [00:00<00:01, 27.46it/s] 18%|################7                                                                            | 9/50 [00:00<00:01, 24.79it/s] 24%|######################                                                                      | 12/50 [00:00<00:01, 23.34it/s] 30%|###########################6                                                                | 15/50 [00:00<00:01, 22.51it/s] 36%|#################################1                                                          | 18/50 [00:00<00:01, 23.73it/s] 42%|######################################6                                                     | 21/50 [00:00<00:01, 25.32it/s] 48%|############################################1                                               | 24/50 [00:00<00:01, 25.48it/s] 54%|#################################################6                                          | 27/50 [00:01<00:00, 25.53it/s] 62%|#########################################################                                   | 31/50 [00:01<00:00, 28.12it/s] 68%|##############################################################5                             | 34/50 [00:01<00:00, 28.53it/s] 74%|####################################################################                        | 37/50 [00:01<00:00, 25.39it/s] 84%|#############################################################################2              | 42/50 [00:01<00:00, 30.43it/s] 92%|####################################################################################6       | 46/50 [00:01<00:00, 26.99it/s] 98%|##########################################################################################1 | 49/50 [00:01<00:00, 26.63it/s]100%|############################################################################################| 50/50 [00:01<00:00, 26.20it/s]
  0%|                                                                                                     | 0/13 [00:00<?, ?it/s] 23%|#####################4                                                                       | 3/13 [00:00<00:00, 25.63it/s] 46%|##########################################9                                                  | 6/13 [00:00<00:00, 23.37it/s] 69%|################################################################3                            | 9/13 [00:00<00:00, 22.83it/s] 92%|####################################################################################9       | 12/13 [00:00<00:00, 23.85it/s]100%|############################################################################################| 13/13 [00:00<00:00, 23.36it/s]
