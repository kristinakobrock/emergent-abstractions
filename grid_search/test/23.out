# ["--attributes=3", "--values=4", "--game_size=10", "--vocab_size_factor=3", "--n_epochs=60", "--batch_size=128", "--learning_rate=0.001", "--hidden_size=256", "--temp_update=0.99", "--temperature=2"]
Creating train_ds and val_ds...

Creating test_ds...
{"loss": 0.7720125913619995, "acc": 0.5311923623085022, "length": 3.444042444229126, "mode": "train", "epoch": 1}
{"loss": 0.6734828948974609, "acc": 0.6357422471046448, "length": 1.0, "mode": "test", "epoch": 1}
{"loss": 0.7052832245826721, "acc": 0.611059308052063, "length": 2.267268657684326, "mode": "train", "epoch": 2}
{"loss": 0.7033822536468506, "acc": 0.49375003576278687, "length": 1.0, "mode": "test", "epoch": 2}
{"loss": 0.6770659685134888, "acc": 0.5905734300613403, "length": 1.1850985288619995, "mode": "train", "epoch": 3}
{"loss": 0.6728124022483826, "acc": 0.6357422471046448, "length": 1.0, "mode": "test", "epoch": 3}
{"loss": 0.6680836081504822, "acc": 0.621208906173706, "length": 1.1159656047821045, "mode": "train", "epoch": 4}
{"loss": 0.6664941310882568, "acc": 0.6195312142372131, "length": 1.0, "mode": "test", "epoch": 4}
{"loss": 0.6642811894416809, "acc": 0.6219570636749268, "length": 1.1195553541183472, "mode": "train", "epoch": 5}
{"loss": 0.6584876179695129, "acc": 0.6357422471046448, "length": 1.0, "mode": "test", "epoch": 5}
{"loss": 0.6609999537467957, "acc": 0.6237820982933044, "length": 1.118347406387329, "mode": "train", "epoch": 6}
{"loss": 0.658171534538269, "acc": 0.6343750357627869, "length": 1.0, "mode": "test", "epoch": 6}
{"loss": 0.6600131392478943, "acc": 0.6248124837875366, "length": 1.1121889352798462, "mode": "train", "epoch": 7}
{"loss": 0.6577224731445312, "acc": 0.634765625, "length": 1.0, "mode": "test", "epoch": 7}
{"loss": 0.6599341630935669, "acc": 0.624697744846344, "length": 1.1013883352279663, "mode": "train", "epoch": 8}
{"loss": 0.6569366455078125, "acc": 0.6357422471046448, "length": 1.0, "mode": "test", "epoch": 8}
{"loss": 0.6595839858055115, "acc": 0.6247052550315857, "length": 1.0946799516677856, "mode": "train", "epoch": 9}
{"loss": 0.6577333211898804, "acc": 0.6314453482627869, "length": 1.0, "mode": "test", "epoch": 9}
{"loss": 0.6586547493934631, "acc": 0.6252456903457642, "length": 1.0865648984909058, "mode": "train", "epoch": 10}
{"loss": 0.6571816205978394, "acc": 0.634765625, "length": 1.0, "mode": "test", "epoch": 10}
{"loss": 0.6590501070022583, "acc": 0.6249207854270935, "length": 1.0873968601226807, "mode": "train", "epoch": 11}
{"loss": 0.657471776008606, "acc": 0.634960949420929, "length": 1.0, "mode": "test", "epoch": 11}
{"loss": 0.6574603319168091, "acc": 0.6251341104507446, "length": 1.083979845046997, "mode": "train", "epoch": 12}
{"loss": 0.6577032804489136, "acc": 0.6339845061302185, "length": 1.0, "mode": "test", "epoch": 12}
{"loss": 0.6580252647399902, "acc": 0.6251342296600342, "length": 1.094376802444458, "mode": "train", "epoch": 13}
{"loss": 0.6578526496887207, "acc": 0.633007824420929, "length": 1.0, "mode": "test", "epoch": 13}
{"loss": 0.6579378247261047, "acc": 0.6255274415016174, "length": 1.1351531744003296, "mode": "train", "epoch": 14}
{"loss": 0.6585434079170227, "acc": 0.6322265267372131, "length": 1.0, "mode": "test", "epoch": 14}
{"loss": 0.6531146764755249, "acc": 0.627163827419281, "length": 1.345971941947937, "mode": "train", "epoch": 15}
{"loss": 0.6426385045051575, "acc": 0.6465820670127869, "length": 1.46875, "mode": "test", "epoch": 15}
{"loss": 0.6369329690933228, "acc": 0.6405349373817444, "length": 1.8249728679656982, "mode": "train", "epoch": 16}
{"loss": 0.6317201852798462, "acc": 0.6496002078056335, "length": 2.1484375, "mode": "test", "epoch": 16}
{"loss": 0.6237679719924927, "acc": 0.6553288102149963, "length": 2.1814193725585938, "mode": "train", "epoch": 17}
{"loss": 0.6408777236938477, "acc": 0.6398299336433411, "length": 2.11328125, "mode": "test", "epoch": 17}
{"loss": 0.6115039587020874, "acc": 0.667205810546875, "length": 2.0870401859283447, "mode": "train", "epoch": 18}
{"loss": 0.6338621377944946, "acc": 0.6469284892082214, "length": 1.99609375, "mode": "test", "epoch": 18}
{"loss": 0.6100844740867615, "acc": 0.6698130965232849, "length": 1.9551596641540527, "mode": "train", "epoch": 19}
{"loss": 0.6458333730697632, "acc": 0.6326842904090881, "length": 2.21875, "mode": "test", "epoch": 19}
{"loss": 0.6072076559066772, "acc": 0.6723172664642334, "length": 2.1335349082946777, "mode": "train", "epoch": 20}
{"loss": 0.6443808078765869, "acc": 0.6354827880859375, "length": 2.06640625, "mode": "test", "epoch": 20}
{"loss": 0.603078305721283, "acc": 0.6741395592689514, "length": 2.3584818840026855, "mode": "train", "epoch": 21}
{"loss": 0.6439156532287598, "acc": 0.642852783203125, "length": 2.25390625, "mode": "test", "epoch": 21}
{"loss": 0.5976938605308533, "acc": 0.6798948049545288, "length": 2.3351075649261475, "mode": "train", "epoch": 22}
{"loss": 0.6413609981536865, "acc": 0.646331787109375, "length": 2.3125, "mode": "test", "epoch": 22}
{"loss": 0.5924525856971741, "acc": 0.6878446340560913, "length": 2.4971160888671875, "mode": "train", "epoch": 23}
{"loss": 0.6163458824157715, "acc": 0.6766861081123352, "length": 2.875, "mode": "test", "epoch": 23}
{"loss": 0.5849609375, "acc": 0.69365394115448, "length": 2.7585105895996094, "mode": "train", "epoch": 24}
{"loss": 0.6022504568099976, "acc": 0.6890106201171875, "length": 3.0859375, "mode": "test", "epoch": 24}
{"loss": 0.5752992630004883, "acc": 0.6987180113792419, "length": 2.9421095848083496, "mode": "train", "epoch": 25}
{"loss": 0.5922490358352661, "acc": 0.6986480951309204, "length": 3.0390625, "mode": "test", "epoch": 25}
{"loss": 0.5682317614555359, "acc": 0.7041909694671631, "length": 3.0185089111328125, "mode": "train", "epoch": 26}
{"loss": 0.5895163416862488, "acc": 0.6983277201652527, "length": 3.0390625, "mode": "test", "epoch": 26}
{"loss": 0.564971387386322, "acc": 0.7058365345001221, "length": 3.027465581893921, "mode": "train", "epoch": 27}
{"loss": 0.5840084552764893, "acc": 0.7007080316543579, "length": 3.02734375, "mode": "test", "epoch": 27}
{"loss": 0.561540424823761, "acc": 0.7075894474983215, "length": 3.015903949737549, "mode": "train", "epoch": 28}
{"loss": 0.5850646495819092, "acc": 0.6941909790039062, "length": 3.00390625, "mode": "test", "epoch": 28}
{"loss": 0.5569496750831604, "acc": 0.7115628123283386, "length": 3.0186173915863037, "mode": "train", "epoch": 29}
{"loss": 0.579407811164856, "acc": 0.7076659202575684, "length": 3.0390625, "mode": "test", "epoch": 29}
{"loss": 0.5531601309776306, "acc": 0.7131145000457764, "length": 3.0296308994293213, "mode": "train", "epoch": 30}
{"loss": 0.5863118171691895, "acc": 0.7058380842208862, "length": 3.02734375, "mode": "test", "epoch": 30}
{"loss": 0.545709490776062, "acc": 0.7188937664031982, "length": 3.046353816986084, "mode": "train", "epoch": 31}
{"loss": 0.5983266830444336, "acc": 0.6909332871437073, "length": 3.02734375, "mode": "test", "epoch": 31}
{"loss": 0.536627471446991, "acc": 0.7319786548614502, "length": 3.0406832695007324, "mode": "train", "epoch": 32}
{"loss": 0.5943867564201355, "acc": 0.7131668329238892, "length": 2.95703125, "mode": "test", "epoch": 32}
{"loss": 0.5202377438545227, "acc": 0.7477051615715027, "length": 3.0183019638061523, "mode": "train", "epoch": 33}
{"loss": 0.5958672761917114, "acc": 0.7191468477249146, "length": 3.015625, "mode": "test", "epoch": 33}
{"loss": 0.5103944540023804, "acc": 0.7576474547386169, "length": 3.0710949897766113, "mode": "train", "epoch": 34}
{"loss": 0.5839887857437134, "acc": 0.7278351187705994, "length": 2.9453125, "mode": "test", "epoch": 34}
{"loss": 0.49757882952690125, "acc": 0.7636032700538635, "length": 3.0423684120178223, "mode": "train", "epoch": 35}
{"loss": 0.5672314167022705, "acc": 0.7253463864326477, "length": 2.98046875, "mode": "test", "epoch": 35}
{"loss": 0.49199962615966797, "acc": 0.7687745690345764, "length": 3.0901753902435303, "mode": "train", "epoch": 36}
{"loss": 0.5567889213562012, "acc": 0.7254517078399658, "length": 3.0390625, "mode": "test", "epoch": 36}
{"loss": 0.48540207743644714, "acc": 0.7700013518333435, "length": 3.1049628257751465, "mode": "train", "epoch": 37}
{"loss": 0.5649688243865967, "acc": 0.72076416015625, "length": 3.09765625, "mode": "test", "epoch": 37}
{"loss": 0.4776325523853302, "acc": 0.7707167863845825, "length": 3.162405490875244, "mode": "train", "epoch": 38}
{"loss": 0.5587083101272583, "acc": 0.7314361333847046, "length": 3.0859375, "mode": "test", "epoch": 38}
{"loss": 0.47324612736701965, "acc": 0.7711518406867981, "length": 3.1581177711486816, "mode": "train", "epoch": 39}
{"loss": 0.5494673252105713, "acc": 0.7401748895645142, "length": 3.05078125, "mode": "test", "epoch": 39}
{"loss": 0.4678284823894501, "acc": 0.7783082723617554, "length": 3.1411118507385254, "mode": "train", "epoch": 40}
{"loss": 0.5501415729522705, "acc": 0.7417587041854858, "length": 3.09765625, "mode": "test", "epoch": 40}
{"loss": 0.46089109778404236, "acc": 0.7776609063148499, "length": 3.276127815246582, "mode": "train", "epoch": 41}
{"loss": 0.5540149807929993, "acc": 0.7336913347244263, "length": 3.25, "mode": "test", "epoch": 41}
{"loss": 0.45441994071006775, "acc": 0.7803245186805725, "length": 3.3925390243530273, "mode": "train", "epoch": 42}
{"loss": 0.5267993211746216, "acc": 0.7389571070671082, "length": 3.671875, "mode": "test", "epoch": 42}
{"loss": 0.44611480832099915, "acc": 0.7829147577285767, "length": 3.575568199157715, "mode": "train", "epoch": 43}
{"loss": 0.5315083265304565, "acc": 0.7396301031112671, "length": 3.75390625, "mode": "test", "epoch": 43}
{"loss": 0.44016364216804504, "acc": 0.7859047651290894, "length": 3.608565092086792, "mode": "train", "epoch": 44}
{"loss": 0.5243805646896362, "acc": 0.7523697018623352, "length": 3.8828125, "mode": "test", "epoch": 44}
{"loss": 0.4385626018047333, "acc": 0.7893224954605103, "length": 3.690546751022339, "mode": "train", "epoch": 45}
{"loss": 0.5364841222763062, "acc": 0.7430282235145569, "length": 3.8828125, "mode": "test", "epoch": 45}
{"loss": 0.4290408790111542, "acc": 0.7925673723220825, "length": 3.710444450378418, "mode": "train", "epoch": 46}
{"loss": 0.5249358415603638, "acc": 0.747467041015625, "length": 3.82421875, "mode": "test", "epoch": 46}
{"loss": 0.42401787638664246, "acc": 0.7975536584854126, "length": 3.729511260986328, "mode": "train", "epoch": 47}
{"loss": 0.5276756286621094, "acc": 0.7473236322402954, "length": 3.8828125, "mode": "test", "epoch": 47}
{"loss": 0.41962677240371704, "acc": 0.797776460647583, "length": 3.754781484603882, "mode": "train", "epoch": 48}
{"loss": 0.5371462106704712, "acc": 0.7446991205215454, "length": 3.87109375, "mode": "test", "epoch": 48}
{"loss": 0.41414695978164673, "acc": 0.8045059442520142, "length": 3.7523176670074463, "mode": "train", "epoch": 49}
{"loss": 0.5243974924087524, "acc": 0.7457442879676819, "length": 3.8671875, "mode": "test", "epoch": 49}
{"loss": 0.41204625368118286, "acc": 0.8015654683113098, "length": 3.7832958698272705, "mode": "train", "epoch": 50}
{"loss": 0.5319652557373047, "acc": 0.7449097037315369, "length": 3.890625, "mode": "test", "epoch": 50}
{"loss": 0.4018559157848358, "acc": 0.810038149356842, "length": 3.793822765350342, "mode": "train", "epoch": 51}
{"loss": 0.5245606899261475, "acc": 0.7595824599266052, "length": 3.96484375, "mode": "test", "epoch": 51}
{"loss": 0.39984655380249023, "acc": 0.8126806616783142, "length": 3.8466038703918457, "mode": "train", "epoch": 52}
{"loss": 0.5291603803634644, "acc": 0.7546752691268921, "length": 3.91796875, "mode": "test", "epoch": 52}
{"loss": 0.3968580663204193, "acc": 0.8110199570655823, "length": 3.816990852355957, "mode": "train", "epoch": 53}
{"loss": 0.5248100757598877, "acc": 0.7511382699012756, "length": 3.9609375, "mode": "test", "epoch": 53}
{"loss": 0.3862905502319336, "acc": 0.8152320384979248, "length": 3.8451385498046875, "mode": "train", "epoch": 54}
{"loss": 0.5231298208236694, "acc": 0.7570556402206421, "length": 3.9609375, "mode": "test", "epoch": 54}
{"loss": 0.3809264004230499, "acc": 0.8240781426429749, "length": 3.8760881423950195, "mode": "train", "epoch": 55}
{"loss": 0.5175957679748535, "acc": 0.7715240120887756, "length": 3.94921875, "mode": "test", "epoch": 55}
{"loss": 0.36555641889572144, "acc": 0.8338382244110107, "length": 3.8650496006011963, "mode": "train", "epoch": 56}
{"loss": 0.4985913038253784, "acc": 0.7765839099884033, "length": 3.9609375, "mode": "test", "epoch": 56}
{"loss": 0.35703203082084656, "acc": 0.8401721715927124, "length": 3.8804433345794678, "mode": "train", "epoch": 57}
{"loss": 0.5179574489593506, "acc": 0.7838563323020935, "length": 3.96484375, "mode": "test", "epoch": 57}
{"loss": 0.3450261056423187, "acc": 0.8489065766334534, "length": 3.8869314193725586, "mode": "train", "epoch": 58}
{"loss": 0.48703819513320923, "acc": 0.8011245727539062, "length": 3.9453125, "mode": "test", "epoch": 58}
{"loss": 0.3363044857978821, "acc": 0.855597734451294, "length": 3.9233713150024414, "mode": "train", "epoch": 59}
{"loss": 0.4825882315635681, "acc": 0.80047607421875, "length": 3.95703125, "mode": "test", "epoch": 59}
{"loss": 0.3269267678260803, "acc": 0.8602911829948425, "length": 3.941138982772827, "mode": "train", "epoch": 60}
{"loss": 0.47264716029167175, "acc": 0.803924560546875, "length": 3.953125, "mode": "test", "epoch": 60}
test accuracy: 0.8200191855430603
